src/prompt/type_prompt.rs

```diff
@@ -121,12 +123,12 @@ impl<'a> TypePrompt<'a> {
                 style(s).with(Color::Magenta).to_string()
             });
 
-            let y_offset = header.len() as u16 + 1;
-
             for line in header {
                 buffer.push_line(line);
             }
 
+            let y_offset = buffer.lines() + 1;
+
```

Comment:
This is just a consistency thing. `y_offset` is determined by `buffer.lines()` on the files prompt and it needs to be as it's the header + a few lines. I figure there's no reason not to do the same here. I don't think it's any slower and consistency is always nice.